tools_for_todoist/models/google_calendar.py

```diff
@@ -93,7 +93,7 @@ def _process_sync(self):
             recurring_event_id = event.get('recurringEventId')
             if recurring_event_id is not None:
                 pending_exceptions.append(event)
-            elif event['status'] == 'cancelled':
+            elif event['status'] == 'cancelled' or is_declined_by_me(event):
```

Comment:
I don't think this is a good idea, we should keep tracking the event and not exclude it from the memory state. This is an update for all intents and purposes.

Reply:
Maybe for single events we should complete it when you decline the event, this way if you accept it later we can unarchive the todoist item instead of creating a new one?
For recurring events declining a repetition should just be tracked, and `next_occurence` should take it into consideration